src/fields/field_fe.hh

```diff
@@ -21,11 +21,18 @@
 #include "petscmat.h"
 #include "system/system.hh"
 #include "fields/field_algo_base.hh"
+#include "fields/vec_seq_double.hh"
+#include "mesh/mesh.h"
 #include "mesh/point.hh"
+#include "mesh/bih_tree.hh"
+#include "mesh/ngh/include/ngh_interface.hh"
```

Comment:
Remove NGH dependency.
Try to minimize includes, replace by forward declarations where possible.